pkg/clusteragent/autoscaling/workload/controller_vertical.go

```diff
@@ -69,9 +69,32 @@ func (u *verticalController) sync(ctx context.Context, podAutoscaler *datadoghq.
 	targetGVK, err := autoscalerInternal.TargetGVK()
 	if err != nil {
 		autoscalerInternal.SetError(err)
+		for _, resource := range scalingValues.Vertical.ContainerResources {
+			telemetryVerticalScale.Inc(
```

Comment:
Same here, we want a `Gauge` with target value being the the limit or request value.